### PR TITLE
[stable/redmine] adds configurable liveness and readiness probes

### PR DESCRIPTION
**Pull request by ErmakovDmitriy**
Signed-off-by: Dmitrii Ermakov <demonihin@gmail.com>

#### What this PR does / why we need it:
PR allows configuring liveness and readiness probes for stable/redmine.

I had a problem:
liveness initialDelaySeconds: 300 was not enough for my environment to load so container continuously restarted.
I had to set the delay to 600 seconds for my environment.

I hope adding configurable timings might help in case of slow environment.
Default values for probes remain the same.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/redmine/values.yaml [hunk 1]
```diff
@@ -297,3 +297,23 @@ resources: {}
...
+
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes)
+## Configure extra options for readiness probe
+readinessProbe:
+  enabled: true
+  initialDelaySeconds: 5
+  periodSeconds: 10
+  timeoutSeconds: 1
+  successThreshold: 1
+  failureThreshold: 3
```
Comment (alemorcuq) [inline stable/redmine/values.yaml]: Could you adapt this to the same format we are using in the rest of the file?

```yaml
## Configure extra options for liveness probe
## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes)
##
livenessProbe:
  enabled: true
  initialDelaySeconds: 300
  periodSeconds: 10
  timeoutSeconds: 5
  successThreshold: 1
  failureThreshold: 3

## Configure extra options for readiness probe
## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes)
##
readinessProbe:
  enabled: true
  initialDelaySeconds: 5
  periodSeconds: 10
  timeoutSeconds: 1
  successThreshold: 1
  failureThreshold: 3
```

**Review by alemorcuq — CHANGES_REQUESTED** [review]
Thanks for the PR, @ErmakovDmitriy, this is a much needed feature. Please, increase the chart version and document the new parameters in the `README.md` file.

**Comment by alemorcuq** [comment]
The DCO is failing, so remember to sign your commits. Thanks!

File: stable/redmine/values.yaml (hunk 1 above)
Comment (ErmakovDmitriy) [reply stable/redmine/values.yaml]: Yes, thank you!
I hope I fixed it.

**Comment by ErmakovDmitriy** [comment]
My request fails "version bump". Could you give me advice if I should change version from 13.0.1 to 13.0.2 or 13.1.0?

**Comment by alemorcuq** [comment]
> My request fails "version bump". Could you give me advice if I should change version from 13.0.1 to 13.0.2 or 13.1.0?

I'd say a minor since you are implementing a new feature. Thanks!

**Review by alemorcuq — APPROVED** [review]
Thank you very much for this, @ErmakovDmitriy. LGTM.

**Comment by alemorcuq** [comment]
@carrodher, please, start the merging process.

**Comment by carrodher** [comment]
There are some conflicts, happy to add the proper tags once resolved.

**Comment by ErmakovDmitriy** [comment]
@carrodher thank you.
I hope I fixed the conflicts.

**Pull request merged**